pegasus/sites.v3/code.org/homepage.json

```diff
@@ -27,38 +27,6 @@
         }
       ]
     },
-    "hoc2023-actual-hoc": {
```

Comment:
Removing last year's banner since I'm not using this json setup for this banner.